uint8 system_id
uint8 component_id
string timestamp
uint8 severity_level
string severity
string text
